suite-native/intl/src/localeSlice.ts

```diff
@@ -28,7 +28,20 @@ export const localeSlice = createSlice({
     },
 });
 
-export const selectUserSelectedLocaleTag = (state: LocaleSliceRootState) => state.locale.localeTag;
-
 export const { setLocale } = localeSlice.actions;
 export const localeReducer = localeSlice.reducer;
+
+export const selectUserSelectedLocaleTag = (state: LocaleSliceRootState) => state.locale.localeTag;
+
+export const selectIsLanguageLocalizationSupported = (
```

Comment:
I'd find this (or similar) more accurate:
```suggestion
export const selectIsLocaleSupported = (
```

Reply:
I believe that `selectIsLanguageLocalizationSupported` is more accurate, because it is talking about language translations localization only. We are also localizing other things in the app apart from the translations as dates and numbers. 

There can be case where user has e.g. `es-ES` (spanish) system locale. In that case `selectIsLanguageLocalizationSupported` will return `false` because we do not have spanish translations yet, but the number and date will be correctly formated according to the `es-ES` locale.

Reply:
OK, what about `selectIsLanguageLocaleSupported` then? (Or maybe `selectIsLocaleLanguageSupported`.)

Reply:
Ok, why not. Fixed and squashed.